common/src/resource.rs

```diff
@@ -250,7 +254,7 @@ impl Display for Type {
 // this can be removed when deployers AND r-r no longer hold resources in sqlite state
 #[cfg(feature = "sqlx")]
 mod _sqlx {
```

Comment:
FYI This module is getting :bomb: in #1962

Reply:
🎉